flow-typed/npm/@emotion/is-prop-valid_vx.x.x.js

```diff
@@ -0,0 +1,3 @@
+declare module '@emotion/is-prop-valid' {
```

Comment:
the package is written using flow & ships a siblings files to entry points: `dist/is-prop-valid.cjs.js.flow` & `is-prop-valid.esm.js.flow`

flow should pick those typings automatically 🤔 hasnt it?

Reply:
It was raising a module not found error... dunno

Reply:
Thanks for the info, need to investigate it later

Reply:
You’re probably ignoring everything in node_modules. You might want to use the new [declarations option](https://flow.org/en/docs/config/declarations/) in flow which ignores errors in the specified folders but still uses the type declarations. (You might be on an older version of flow that doesn’t support it though so you might need to update flow)